cmd/argo/commands/common.go

```diff
@@ -94,8 +95,8 @@ func InitWorkflowClient(ns ...string) v1alpha1.WorkflowInterface {
 			log.Fatal(err)
 		}
 	}
```

Comment:
Why not keep the origin code ?

Reply:
It seemed confusing. I can revert the names if that is a problem